### ISSUE TITLE
Roadmap to 1.0?

### ISSUE THREAD

**Issue by 11reed** [mattjennings/svelte-phaser#20]
I am not sure if this is still maintained, but I was wondering if there is a possibility for a roadmap.

**Comment by mattjennings** [comment]
Unfortunately I am not focused on this project right now. I would like to come back to it at some point, but for the time being there’s no roadmap.

**Comment by 11reed** [comment]
Would this be somewhat stable for production use in it's current state? I have a pretty big upcoming project, and I am considering using this.

**Comment by mattjennings** [comment]
No, it's still in what I'd consider a "proof-of-concept" stage and would not be a safe bet for production usage. I'd lean more toward using phaser natively for a big project.

**Comment by 11reed** [comment]
Alright, good to know.
